src/test/java/com/rultor/agents/AgentsITCase.java

```diff
@@ -0,0 +1,768 @@
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.rultor.agents;
+
+import co.stateful.mock.MkSttc;
+import com.google.common.collect.Lists;
+import com.jcabi.github.Assignees;
+import com.jcabi.github.Branches;
+import com.jcabi.github.Collaborators;
+import com.jcabi.github.Comment;
+import com.jcabi.github.Comments;
+import com.jcabi.github.Contents;
+import com.jcabi.github.Coordinates;
+import com.jcabi.github.DeployKeys;
+import com.jcabi.github.Event;
+import com.jcabi.github.Forks;
+import com.jcabi.github.Gists;
+import com.jcabi.github.Git;
+import com.jcabi.github.Github;
+import com.jcabi.github.Gitignores;
+import com.jcabi.github.Hooks;
+import com.jcabi.github.Issue;
+import com.jcabi.github.IssueEvents;
+import com.jcabi.github.IssueLabels;
+import com.jcabi.github.Issues;
+import com.jcabi.github.Labels;
+import com.jcabi.github.Language;
+import com.jcabi.github.Limits;
+import com.jcabi.github.Markdown;
+import com.jcabi.github.Milestones;
+import com.jcabi.github.Notifications;
+import com.jcabi.github.Organizations;
+import com.jcabi.github.Pulls;
+import com.jcabi.github.Releases;
+import com.jcabi.github.Repo;
+import com.jcabi.github.RepoCommits;
+import com.jcabi.github.Repos;
+import com.jcabi.github.Search;
+import com.jcabi.github.Stars;
+import com.jcabi.github.Users;
+import com.jcabi.github.mock.MkBranches;
+import com.jcabi.github.mock.MkGithub;
+import com.jcabi.http.Request;
+import com.jcabi.http.request.FakeRequest;
+import com.jcabi.immutable.ArrayMap;
+import com.jcabi.ssh.SSHD;
+import com.jcabi.xml.XMLDocument;
+import com.rultor.spi.Profile;
+import com.rultor.spi.Talk;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+import java.util.ResourceBundle;
+import javax.json.Json;
+import javax.json.JsonObject;
+import javax.json.JsonObjectBuilder;
+import javax.json.JsonValue;
+import org.apache.commons.lang3.SystemUtils;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Assume;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.mockito.Mockito;
+import org.xembly.Directives;
+
+/**
+ * Integration test for ${@link Agents}.
+ *
+ * @author Denys Skalenko (d.skalenko@gmail.com)
+ * @version $Id$
+ * @since 2.0
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 line)
+ */
+@SuppressWarnings("all")
```

Comment:
@dskalenko this is not acceptable. Please only suppress the warnings you need to suppress, preferable none!
If you need to suppress any, please as in the above lay out the reasons in the PR description.